BlueprintUI/Sources/Measuring/RoundingScale.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+/// Represents a scale rounding behavior for `CGFloat`s.
+public enum RoundingScale: Equatable {
+    /// Do not round
+    case none
```

Comment:
Why is the `none` case included?

Reply:
See @bencochran's [comment suggesting this previously](https://github.com/square/Blueprint/pull/18#discussion_r334164836)

Reply:
Ah – I see where that is coming from.

The name of the type (`RoundingScale`) currently implies that it represents a screen scale for use when rounding. In that case, `.none` seems unexpected.  I could instead just..... not call the rounding method.

It seems based on the discussion that this type is meant to model **the desired rounding behavior of an element**: so I would name it as such. `PixelRoundingBehavior` or something. `.none` is a very reasonable *behavior*, but it isn't a very useful *screen scale*.

Reply:
I would also add the use of this type to the documentation: it isn't a general purpose rounding tool, it's a way to specify a rounding behavior for an element